Formula/imagemagick.rb

```diff
@@ -79,19 +83,23 @@ def install
 
     args << "--without-x" if OS.mac?
 
-    # versioned stuff in main tree is pointless for us
-    inreplace "configure", "${PACKAGE_NAME}-${PACKAGE_BASE_VERSION}", "${PACKAGE_NAME}"
+    system "autoreconf", "-fiv"
```

Comment:
Why do we need this?

Reply:
For some reason, the in-tree configure script doesn't correctly detect LibRaw

Reply:
That should probably be fixed upstream then.